pyproject.toml

```diff
@@ -0,0 +1,84 @@
+[tool.black]
+line-length  =  100
+target-version  =  [
+    'py37',
+]
+include  =  '(bluepysnap|tests)\/.*\.py'
```

Comment:
In other projects we include also `doc/source/conf\.py$|setup\.py$`, any reason not to include them?

Reply:
You're right. I'll add them, too.